tests/automation_framework/data/work_order/work_order_submit.yaml

```diff
@@ -177,903 +120,425 @@ test_workordersubmit_indata_bothindexremoveDataDatahash:
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 14
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 13
+    iv: ''
+  - data: 'Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 12
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 11
+    iv: ''
+  - data: 'Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 10
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 9
+    iv: ''
+  - data: 'Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 8
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 7
+    iv: ''
+  - data: 'Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 6
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 5
+    iv: ''
+  - data: 'Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 4
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 3
+    iv: ''
+  - data: 'Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 2
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 1
+    iv: ''
 test_workordersubmit_iv_specialcharacterechoclient:
   iv: abcd@123
 test_workordersubmit_mandatoryfields_remove:
   workOrderId: remove
   workerId: remove
   workloadId: remove
 test_workordersubmit_methodname_list:
-  method: '["WorkOrderSubmit","WorkOrderSubmit"]'
+  method:
+  - WorkOrderSubmit
+  - WorkOrderSubmit
 test_workordersubmit_outdata_empty:
-  params: '{"outData": []}'
+  params:
+    outData: []
 test_workordersubmit_outdata_success:
-  params: '{
-
-    "outData": [
-
-    {"index" : 0,
-
-    "data": "",
-
-    "encryptedDataEncryptionKey": "",
-
-    "iv": ""
-
-    }
-
-    ]}'
+  params:
+    outData:
+    - data: ''
+      encryptedDataEncryptionKey: ''
+      index: 0
+      iv: ''
+test_workordersubmit_params_empty:
```

Comment:
For this tests "params :{}" should also work

Reply:
I just tried it. It didn't work. still took all the default values. Keeping it the same for now.